Formula/gcc.rb

```diff
@@ -111,12 +111,14 @@ def install
       args << "--build=#{cpu}-apple-darwin#{OS.kernel_version.major}"
       args << "--with-system-zlib"
 
-      # Xcode 10 dropped 32-bit support
-      args << "--disable-multilib" if DevelopmentTools.clang_build_version >= 1000
+      # Xcode 10 dropped 32-bit Intel support
+      args << "--disable-multilib" if Hardware::CPU.intel? && DevelopmentTools.clang_build_version >= 1000
 
-      # Workaround for Xcode 12.5 bug on Intel
-      # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100340
-      args << "--without-build-config" if Hardware::CPU.intel? && DevelopmentTools.clang_build_version >= 1205
+        # Workaround for Xcode 12.5 bug on Intel, remove in next version
+        # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100340
+      if Hardware::CPU.intel? && DevelopmentTools.clang_build_version >= 1205 && DevelopmentTools.clang_build_version < 1300
```

Comment:
```suggestion
      if Hardware::CPU.intel? && DevelopmentTools.clang_build_version == 1205
```
Does this work?

Reply:
I honestly have no idea, and can't test all Xcode 12.x versions, so I figured I would play it safe. All I know is it's fixed on the Xcode 13 line ;)

Reply:
I guess my point is that there are no Xcode versions in between `1205` and `1300` AFAIK, so checking for `1205` should be sufficient.